modules/tinymce/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 
 ### Added
+- Added support for `inline_boundaries` to work with home/end/pgup/pgdown keys
```

Comment:
Please include the JIRA number at the end of the sentence like the other entries.

Reply:
```suggestion
- The `inline_boundaries` feature now supports the `home`, `end`, `pageup` and `pgdown` keys #TINY-4612
```

Reply:
updated